ttt-game/src/components/TimeMachine.js

```diff
@@ -0,0 +1,28 @@
+import React, {useState} from 'react'
+
+const TimeMachine = (props) => {
+  
+  const [step, setStep] = useState(0)
+  
+  const deleteParadox = (clickedButton) => {
```

Comment:
I love this function name, but I'm a little bit confused about what this function is doing. Is this erasing part of the history when you travel back?